dpdata/system.py

```diff
@@ -806,3 +858,11 @@ def check_LabeledSystem(data):
        assert( len(data['cells']) == len(data['coords']) == len(data['energies']) )
 
 
+def elements_index_map(elements,standard=False,inverse=False):
```

Comment:
It is highly recommended to add documents that explain the parameters **standard** and **inverse**